content/en/tutorials/migrate_gcp_122.md

```diff
@@ -0,0 +1,18 @@
+# Upgrading Google Kubernetes Engine Version
+
+The Kubernetes Project releases minor version updates every quarter, and users are expected to upfate their infrastructure at least within a few minor versions of the latest Kubernetes release. With Google GKE, Opta relies on GCP's [release channels](https://cloud.google.com/kubernetes-engine/docs/concepts/release-channels) to automatically upgrade the Kubernetes control plane as well as the Kubernetes worker nodes. When Opta installs your GKE cluster, it uses the "regular" GKE channel. Currently (Feb 1, 2022), this channel will install a Kubernetes 1.21.x cluster.
+
+In the next days the GKE regular channel will start updating Kubernetes to 1.22.x; which will mean that your opta-deployed Kubernetes cluster is automatically upgraded to a 1.22.x version. This version of Kubernetes replaces several deprecated Kubernetes APIs and replaces them with newer versions. So there are two aspects we need to proactively manage.
```

Comment:
_In the next days_ we can say something like `GKE would start start updating Kubernetes to 1.22.x a few days after this date (Feb 1, 2022 - mentioned before)